README.md

```diff
@@ -51,14 +50,171 @@ Read in a different language: [![CN](./assets/flags/CN.png)**CN**](./README.chin
 
 ## Table of Contents
 
-1. [Project Structure Practices (5)](#1-project-structure-practices)
-2. [Error Handling Practices (12) ](#2-error-handling-practices)
-3. [Code Style Practices (12) ](#3-code-style-practices)
-4. [Testing And Overall Quality Practices (13) ](#4-testing-and-overall-quality-practices)
-5. [Going To Production Practices (19) ](#5-going-to-production-practices)
-6. [Security Practices (25)](#6-security-best-practices)
-7. [Performance Practices (2) (Work In Progress️ ✍️)](#7-draft-performance-best-practices)
-8. [Docker Practices (15)](#8-docker-best-practices)
+<details>
+  <summary>
+    <a href="#1-project-structure-practices">1. Project Structure Practices (5)</a>
+  </summary>
+
+&emsp;&emsp;<a href="#-11-structure-your-solution-by-components">1.1 Structure your solution by components</a></br>
+&emsp;&emsp;<a href="#-12-layer-your-components-keep-the-web-layer-within-its-boundaries">1.2 Layer your components, keep the web layer within its boundaries</a></br>
+&emsp;&emsp;<a href="#-13-wrap-common-utilities-as-npm-packages">1.3 Wrap common utilities as npm packages</a></br>
+&emsp;&emsp;<a href="#-14-separate-express-app-and-server">1.4 Separate Express 'app' and 'server'</a></br>
+&emsp;&emsp;<a href="#-15-use-environment-aware-secure-and-hierarchical-config">1.5 Use environment aware, secure and hierarchical config</a></br>
```

Comment:
Can we have the old markup here back, please? We're using MD and not HTML. I've successfully used details/summary with MD inside on GitHub in the past.

Reply:
Look [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) for example

Reply:
I've managed to revert back the sub headers to the old markup but I couldn't get it working for the main header (i.e. inside the `<summary/>` tag).